src/armory/Armory.js

```diff
@@ -281,48 +280,119 @@ class Armory extends React.Component<Props, State> {
 		});
 	};
 
-
+	//The || in the values are for a warning read more here https://github.com/mui-org/material-ui/issues/4904
 	render(): React.Node {
 		return (
 			<div id="Parent">
 				<Navbar styleName="navbtn" linkName="MainMenu" returnName="Back to Main Menu" pageName="Armory" userName="FRIcker | $465128" />
-				<div className="column armoryleft">
-					<h3>Select a Tank to Edit</h3>
-					<select className="dropdownMenu" value={this.state.selectedTankId} onChange={this.handleChangeInTankOptions}>{tankOptions.map(({ value, label }, index) => <option key={index}  value={value}>{label}</option>)}</select>
 					<div className="column armoryleft">
+						<h3>Select a Tank to Edit</h3>
+						<select 
+							className="dropdownMenu" 
+							value={this.state.selectedTankId} 
+							onChange={this.handleChangeInTankOptions}
+						>
+							{tankOptions.map(({ value, label }, index) => <option key={index}  value={value}>{label}</option>)}
```

Comment:
I know this isn't part of the changes you made here (so you don't have to fix it before pushing, just something to keep in mind): the way that this stuff is stored with the value/label/index is pretty confusing